src/libpmemfile-posix/file.c

```diff
@@ -52,6 +53,23 @@
 #include "pool.h"
 #include "util.h"
 
+static int
+vinode_cmp(const void *v1, const void *v2)
+{
+	return (int)((intptr_t)*(void **)v1 - (intptr_t)*(void **)v2);
```

Comment:
Might wanna change this line.
I tried this a little C program, and it printed `0`:
```
int qqq(intptr_t a, intptr_t b)
{
        return (int)(a - b);
}

int main()
{
  intptr_t a = 5;
  intptr_t b = 5 + (((intptr_t)1) << 32);

  printf("qqq: %d\n", qqq(a, b));

}
```